packages/thirdweb/src/universal/Buy.ts

```diff
@@ -0,0 +1,263 @@
+import type { Address as ox__Address } from "ox";
+import type { ThirdwebClient } from "../client/client.js";
+import { getClientFetch } from "../utils/fetch.js";
+import { UNIVERSAL_BRIDGE_URL } from "./constants.js";
+import type { PreparedQuote, Quote } from "./types/Quote.js";
+
+/**
+ * Retrieves a Universal Bridge quote for the provided buy intent. The quote will specify the necessary `originAmount` to receive the desired `destinationAmount`, which is specified with the `buyAmountWei` option.
+ *
+ * @example
+ * ```typescript
+ * import { Universal } from "thirdweb";
+ *
+ * const quote = await Universal.Buy.quote({
```

Comment:
not sure about `Universal` about the entrypoint, but also can't find something i feel much better about 😅 

kinda want to make it more explicit and call the entrypoint the full `UniversalBridge` word

```
import { UniversalBridge } from "thirdweb";

await UniversalBridge.Buy.quote(...)
```
 bit mouthful though.

Reply:
Yeah I feel the same, definitely not set on anything yet

Reply:
Settled on `Bridge`

Reply:
I like bridge, let's go with that